public/app/routes/routes.tsx

```diff
@@ -299,19 +299,15 @@ export function getAppRoutes(): RouteDescriptor[] {
       component: SafeDynamicImport(() => import(/* webpackChunkName: "TeamPages" */ 'app/features/teams/TeamPages')),
     },
     // ADMIN
-    {
-      path: '/admin/authentication',
-      roles: () => contextSrv.evaluatePermission(() => ['Admin', 'ServerAdmin'], [AccessControlAction.SettingsWrite]),
-      component: config.licenseInfo.enabledFeatures?.saml
-        ? SafeDynamicImport(
-            () => import(/* webpackChunkName: "AdminAuthentication" */ 'app/features/auth-config/AuthConfigPage')
-          )
-        : () => <Redirect to="/admin" />,
-    },
     {
       path: '/admin',
       component: () => <NavLandingPage navId="cfg" header={<ConnectionsRedirectNotice />} />,
     },
+    {
```

Comment:
moved it after the /admin route registration.

what is changed is this line:
```go
component: config.licenseInfo.enabledFeatures?.saml
        ? SafeDynamicImport(
            () => import(/* webpackChunkName: "AdminAuthentication" */ 'app/features/auth-config/AuthConfigPage')
          )
        : () => <Redirect to="/admin" />,
```        
to:
```go
component: SafeDynamicImport(() => import(/* webpackChunkName: "AdminAuthentication" */ 'app/features/auth-config/AuthConfigPage')),
```

Reply:
Could you double check what's a behaviour when LDAP is not enabled and edition is OSS (no SAML)? Is it empty page? I guess no CTA should be shown there.